docs/flexibility/trigger-updates.md

```diff
@@ -0,0 +1,58 @@
+
+The value can be a start-up option or changed dynamically.
+
+## Status variables
+
+The following status variables are available:
+<style>
+    table th:first-of-type { width: 40%;
+    }
+    table th:nth-of-type(2) { width: 60%;
+    }
+</style>
+
+| Variable name                       | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+|-------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `table_open_cache_triggers_hits`      | A hit means the statement required an open table instance with fully-loaded triggers and was able to get one from the `table_open_cache`.                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `table_open_cache_triggers_misses`    | A miss means the table requiring an open table instance with fully-loaded triggers was not found in the `table_open_cache`. The variable may find a table instance without fully-loaded triggers.                                                                                                                                                                                                                                                                                                                                                          |
+| `table_open_cache_triggers_overflows` | An overflow indicates the number of unused table instances with triggers were expelled from the `table_open_cache` due to the `table_open_cache_triggers` soft limit. This variable may demonstrate that the `table_open_cache_triggers` value should be increased.
```

Comment:
indicates the number of unused table instances with triggers were expelled -> indicates the number of unused table instances with triggers THAT were expelled

Reply:
updated